### PR TITLE
replication changes

### PR DESCRIPTION
**Pull request by itsmekumari**
replication changes

### CODE REVIEW

**Review by itsankit-google — COMMENTED** [review]
Can you please add a screenshot in the description which will help in determining these locators correspond to which widgets?

**Comment by itsmekumari** [comment]
> Can you please add a screenshot in the description which will help in determining these locators correspond to which widgets?

Screenshot 2023-08-18 at 9 52 54 AM
Screenshot 2023-08-18 at 10 02 19 AM
1.Selecting a created table for replication job  and  2.Verifying error in replication logs

File: src/main/java/io/cdap/e2e/pages/locators/CdfLogLocators.java [hunk 1]
```diff
@@ -60,4 +60,7 @@ public static WebElement locateLogEntry(String logLevel, String logMessage) {
       "//following-sibling::div[@data-cy='log-message'][contains(normalize-space(.), \"" + logMessage + "\")]";
     return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
   }
+
+  @FindBy(xpath = "//div[@data-cy='log-viewer-row']//div[contains(text(),'ERROR')]")
+  public static WebElement logError;
```
Comment (itsankit-google) [inline src/main/java/io/cdap/e2e/pages/locators/CdfLogLocators.java]: can we make it more generic as a method named `locateLogLevel` which takes `logLevel` as an input?

Comment (itsmekumari) [reply src/main/java/io/cdap/e2e/pages/locators/CdfLogLocators.java]: Change done.

**Review by itsankit-google — APPROVED** [review]

**Pull request merged**